packages/admin-ui/src/switch/message.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```

Comment:
Why not use the same radio [message](https://github.com/vtex/admin-ui/blob/main/packages/admin-ui/src/radio/message.tsx)? Like that we can reuse and encapsulate in a FormGroup componente in the future 🤔

Reply:
I would wait for the FormControl to be ready, than change it on demand. Let me know what you think!